public_html/wp-content/mu-plugins/blocks/source/blocks/schedule/schedule-grid.scss

```diff
@@ -0,0 +1,116 @@
+$color__grid-border: #e0e5e9;
+
+/*
+ * todo
+ *
+ * time column is wider than track columns, but it should be equal or maybe a bit smaller
+ *
+ * alignfull causes horizontal scrolling, at least with some themes
+ *
+ * gutenberg's built-in "tablet" and "mobile" doesn't recognize the breakpoints
+ *
+ * setup vars for other colors. use convention from gutenberg if one is documented in future:
+ * https://github.com/WordPress/gutenberg/issues/19352
```

Comment:
Giving you a basic answer to that issue here: Generally, the `$dark-gray-900`, `$dark-gray-800` (vars with numbers) are from the WP (core) color palette, and that's the set of colors you can work with (well, the grays and blues you can use). The numbers correspond to how light/dark they are (higher = darker). The descriptively-named colors _ideally_ would inherit from those, so you'd never create a new grey (though that's not how it works in practice, because there isn't an enforcement of this). Those don't have a naming scheme AFAIK, but we might create one … soon-eventually.

We'd want to use the core colors so that our block's UI will match GB, and so that we don't need to involve a designer just to pick a color. In the past I've cherry-picked some of the base colors down into [a local variables file](https://github.com/WordPress/wordcamp.org/blob/production/public_html/wp-content/plugins/wordcamp-speaker-feedback/assets/scss/_variables.scss) for project use.

Reply:
I don't think this is a blocker for beta, but I appreciate the background info 👍

Reply:
I'm not requesting any changes, was just answering your question.